src/kubectlutils.ts

```diff
@@ -180,3 +180,24 @@ export async function getKubernetesVersion(): Promise<string | undefined> {
 
     return `${major}.${minor}`;
 }
+
+// retrieve the first integration running in camel k starting with the integration name
+export async function getIntegrationsFromKubectl(integrationName : string): Promise<string> {
+	return new Promise <string> ( async (resolve, reject) => {
+		await getPodsFromKubectlCli()
+		.then( (allPods) => {
+			let podArray = parseShellResult(allPods);
+			podArray.forEach(podName => {
+				if (podName.startsWith(integrationName)) {
```

Comment:
Using the first one matching the substring can lead to using the wrong pod, isn't it? Do you know the exact case it can happen?

some other thoughts:
Has the suffix a fixed length? In this case we can substring. I think it would be worthy to handle in this PR in this case (unless we already had the same issue before?)
Otherwise, we might have false positive, maybe we should detect when several are matching and:
- log it as a first iteration in this PR
- propose user to choose the one he wants? This one seems more complicated and might be easier to handle in a separate PR.
- or have another heuristic which is deterministic?

Reply:
Couple of things.

1. This is utility code we have been using for a while to get the pod and retrieve the log for the pod that we used in the output channel. We are not using it any longer (now using kamel log instead of kubectl --logs) but I wanted to keep it in case we needed it in the future. 
2. Because of #1, maybe we should just remove it entirely. 

But to be honest, it has worked great for months with no issues. We're just shifting to a different method of tracking the logs now and don't really need it.

Reply:
And yes, I wish we could test - but without a solid way of integration testing this thing I don't know quite what to do on that front.

Reply:
> 1. but I wanted to keep it in case we needed it in the future.

if we remove it, we will still have it in the history :-)

> 2\. Because of #1, maybe we should just remove it entirely.

I vote for removing it :-)